src/view/reviewCommentController.ts

```diff
@@ -941,17 +903,17 @@ export class ReviewDocumentCommentProvider implements vscode.Disposable, Comment
 
 	// #region Incremental update comments
 	public async update(localFileChanges: GitFileChangeNode[], obsoleteFileChanges: (GitFileChangeNode | RemoteFileChangeNode)[]): Promise<void> {
-		const inDraftMode = await this._prManager.inDraftMode(this._prManager.activePullRequest!);
+		await this._prManager.validateDraftMode(this._prManager.activePullRequest!);
 		// _workspaceFileChangeCommentThreads
 		for (let fileName in this._workspaceFileChangeCommentThreads) {
-			this.updateFileChangeCommentThreads(localFileChanges, fileName, inDraftMode);
+			this.updateFileChangeCommentThreads(localFileChanges, fileName);
```

Comment:
This method is async - should this be awaited too? It might not matter in this case I suppose.